docs/_documentation/platform/android/android-spinner.md

```diff
@@ -80,12 +80,13 @@ Given that we called the Item Template above `country_item_template.axml` it wil
 
 The `HandleItemSelected` command, can be bound to a command in your ViewModel. When executed this will provide the `ViewModel` bound to the selected row as the command parameter.
 
-```csharp
+```c#
```

Comment:
csharp is just as valid as c#, the difference here is that the code block is closed with ```